modules/apps/asset/asset-info-impl/src/main/java/com/liferay/asset/info/internal/item/provider/AssetEntryInfoItemFieldSetProviderImpl.java

```diff
@@ -49,6 +55,32 @@
 public class AssetEntryInfoItemFieldSetProviderImpl
 	implements AssetEntryInfoItemFieldSetProvider {
 
+	@Override
+	public InfoFieldSet getInfoFieldSet(AssetEntry assetEntry) {
```

Comment:
Requiring an AssetEntry instance means that this won't be invocable in contexts where there isn't one such as in a Display Page Template or in a Collection Display. In reality all that it's needed is the assetClassName for global vocabularies and a scopeGroupId for getting the ones of the current group.